.github/workflows/ci.yml

```diff
@@ -617,7 +617,9 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "version=${GITHUB_REF##*/}" | tee --append "$GITHUB_OUTPUT"
+        run: |
+          echo "version=${GITHUB_REF##*/}" | tee --append "$GITHUB_OUTPUT"
+          echo "protocol_version=$(curl -fsSL -H "Authorization: Bearer ${{ github.token }}" https://raw.githubusercontent.com/sass/sass/HEAD/spec/EMBEDDED_PROTOCOL_VERSION)" | tee --append "$GITHUB_OUTPUT"
```

Comment:
Note: `curl -fsSL` is commonly used in scripting:

`-f`: Fail fast with no output at all on server errors.
`-s`: Silent or quiet mode. Do not show progress meter or error messages.
`-S`: When used with -s, --silent, it makes curl show an error message if it fails.
`-L`: If the server reports that the requested page has moved to a different location (indicated with a Location: header and a 3XX response code), this option will make curl redo the request on the new place.